### ISSUE TITLE
THERMAL-Orbital Design Software

### ISSUE THREAD

**Issue by kevvillarreal** [CougsInSpace/CougSat1-Hardware#310]
Gain familiarity with NASA's GMAT software, and its applications in simulating Cougsat 1's space environment.

**Comment by TaylanSwift** [comment]
Update: I found some "Small Sat Tools" outlined by NASA under thermal and radiation analysis. 
Kevin and I got access to Space environment information system (SPENVIS), but it yields obscure graphs and data like electron fluxes, when given orbit altitude, and relevant parameters. Might be useful later on when we know better what "orbital environment" data we need. 
Thermal Desktop was the #1 recc.'d general purpose application - but it' based on AutoCad - which doesn't mesh well with what everyone in CiS is using. 
Then, looking through resources on SolidWorks, it seems that Solid Works Flow Simulation is a decent option for radiation modelling. We'll be investigating how useful it is this week.
